### DISCUSSION TITLE
Does Music Assistant support DSD files?

### DISCUSSION THREAD

**Discussion by solarjunkev** [music-assistant/support#1720] [Q&A] [answered]
Hi Does Music Assistant support the following file formats? 

Stereo .dsf? or .dff
Multichannel .dsf? .dff?
If not is there any capability or plan for file support of DSD in the future? 

Thanks

**Comment by OzGav** [comment]
MA uses ffmpeg so support needs to be added there

**Comment by solarjunkev** [comment]
Ok so the answer is no support at the moment, but it is possible in the future?
Is that correct?
Thanks

**Reply by OzGav** [reply, marked as answer]
As I understand it only if the ffmpeg project adds support. We have no input into that

**Comment by solarjunkev** [comment]
Ok cool.. so clearly it doesn't support native playback of DSD. No problem. Just needed to know (how to arrange my shares) that's all Cheers

**Comment by edisun3003** [comment]
It’s supporting dsd music playback 
Dont know how to enable 5.1 playback

**Reply by OzGav** [reply]
Is this a question? If you are asking how to enable 5.1 playback using Music Assistant then the answer is that isn't possible. MA is stereo only.
